setup.py

```diff
@@ -9,34 +9,33 @@
 
 setup(
     name='django-ses-plus',
-    version='0.2.2',
+    version='1.0.0',
     packages=['django_ses_plus'],
     include_package_data=True,
-    install_requires=['Django >= 1.11', 'django-ses >= 1.0.0', 'celery >= 4'],
+    install_requires=['Django >= 3.2', 'django-ses >= 1.0.0', 'celery >= 4'],
```

Comment:
Let's upgrade required `django-ses` version to `>=2.1.1`. The resaon of `2.1.1` is that they have first started to support Django 3.2 in `2.1.0`, and they have a hotfix in `2.1.1`.

Reply:
Thank you! I forgot about checking the compatibility with `django-ses` .